astroid/nodes/scoped_nodes/scoped_nodes.py

```diff
@@ -1350,12 +1300,12 @@ def __init__(
             frame = parent.frame(future=True)
             frame.set_local(name, self)
 
-    def postinit(
+    def postinit(  # type: ignore[override]
```

Comment:
Why are we not also fixing up the `__init__`?

Reply:
That will require a bit more work. `FunctionDef` isn't in great shape. It has deprecated args, lots unannotated fields, weird extra `__init__` args. Plus this `Lambda` inheritance problem.